conans/test/integration/command/download/download_parallel_test.py

```diff
@@ -11,15 +11,14 @@ def test_basic_parallel_download():
 
     package_ids = []
     for i in range(counter):
-        client.run("create . --name=pkg --version=0.1 --user=user --channel=testing -o pkg/*:myoption=%s" % i)
-        package_id = client.created_package_id("pkg/0.1@user/testing")
-        package_ids.append(package_id)
+        client.run(f"create . --name=pkg{i} --version=0.1 --user=user --channel=testing -o pkg{i}/*:myoption={i}")
+        package_id = client.created_package_id(f"pkg{i}/0.1@user/testing")
+        package_ids.append((i, package_id))
```

Comment:
This test was covering multiple binaries for the same pkg/version. With this change, it will test multiple pkgi/version with one single binary for each one. Maybe we want to cover both cases, adding some binaries for the same package name too.